include/starsh-spatial.h

```diff
@@ -48,6 +48,13 @@ typedef struct starsh_ssdata
     //!< Noise and regularization parameter.
     double sigma;
     //!< Variance.
+    double nu2;
+    //!< Order of Mat&eacute;rn kernel.
+    double noise2;
```

Comment:
seems like this parameter is unused

Reply:
Yes, you are correct. I removed it for now.